service-admin/internal/server/handlers/search.go

```diff
@@ -67,9 +67,10 @@ type Search struct {
 }
 
 type AddedBy struct {
-	DateAdded  string
-	Email      string
-	ActivateBy int
+	DateAdded     string
+	Email         string
+	ActivateBy    int
+	ActivatedOn string
```

Comment:
```suggestion
	DateAdded   string
	Email       string
	ActivateBy  int
	ActivatedOn string
```

This should have been fixed by go-fmt. Which in most IDE's runs on document saving. Please check this is setup.